dataikuapi/dss/scenario.py

```diff
@@ -338,32 +339,59 @@ def raw_reporters(self):
 
     def add_periodic_trigger(self, every_minutes=5):
         """Adds a trigger that runs the scenario every X minutes"""
-        trigger = {"active": True, "type": "temporal", "params": { "frequency": "Minutely", "count": every_minutes}}
+        trigger = {"active": True, "type": "temporal", "params": { "frequency": "Minutely", "repeatFrequency": every_minutes }}
         self.raw_triggers.append(trigger)
 
-    def add_hourly_trigger(self, minute_of_hour=0):
+    def add_hourly_trigger(self, minute_of_hour=0, year=None, month=None, day=None, starting_hour=0, repeat_every=1, timezone="SERVER"):
         """Adds a trigger that runs the scenario each hour on a predefined minute"""
-        trigger = {"active": True, "type": "temporal", "params": { "frequency": "Hourly", "minute":  minute_of_hour}}
+        starting_date =self.__get_starting_date__(year=year, month=month, day=day)
+        trigger = {"active": True, "type": "temporal", "params": { "frequency": "Hourly", "hour": starting_hour, "minute": minute_of_hour,
+                                                                   "startingFrom": starting_date, "repeatFrequency": repeat_every, "timezone": timezone }}
         self.raw_triggers.append(trigger)
 
-    def add_daily_trigger(self, hour=2, minute=0, days=None):
+    def add_daily_trigger(self, hour=2, minute=0, days=None, year=None, month=None, day=None, repeat_every=1, timezone="SERVER"):
```

Comment:
I hate this paramter ordering, but we have to keep the legacy working...
If you have a better ordering that is consistent between the 4 methods, I would gladly get it.